html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About Me</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="../about-me/about-me.html">About Me</a></li>
```

Comment:
Since we're already in the `about-me` folder, the `../` isn't necessary. We don't need to go up a level just to go back down. You can just do `href="about-me.html"`